doc/api/modules.md

```diff
@@ -22,9 +22,9 @@ Here are the contents of `circle.js`:
 ```js
 const { PI } = Math;
 
-exports.area = (r) => PI * r * r;
+exports.area = r => PI * r * r;
```

Comment:
@lpinca  There is a nuance here. I mean somethin like [this advice](https://github.com/airbnb/javascript#arrows--implicit-return): when we have implicite return and the function body is not in braces, omit parentheses around one argument. I've searched the https://nodejs.org/api/all.html and have found only 2 such cases, and they are better be rewritten with braces, because they do not use returned value and have side effects:

https://github.com/nodejs/node/blame/47f8f7462fb198aa27ede602c43786bdbfda37a2/doc/api/process.md#L313

https://github.com/nodejs/node/blame/82ef00cc0a36492808badb23d4d79ed303412943/doc/api/stream.md#L1757

Then I've searched the core libs and find these cases and have found only [one fragment](https://github.com/nodejs/node/blob/2e568d95bdd689494b163f1cfe8bbc38f32e45ed/lib/punycode.js#L133) with omitted parentheses and dozens with them. So I shall revert this cange to conform with core.